Classes/Service/AuthenticationService.php

```diff
@@ -285,7 +285,8 @@ protected function convertResourceOwner(array $info)
             ? (new \TYPO3\CMS\Core\Information\Typo3Version())->getBranch()
             : TYPO3_branch;
         if (version_compare($typo3Branch, '9.0', '>=')) {
-            $objInstanceSaltedPW = \TYPO3\CMS\Saltedpasswords\Salt\SaltFactory::getDefaultHashInstance(TYPO3_MODE);
+            $saltFactory = GeneralUtility::makeInstance(\TYPO3\CMS\Saltedpasswords\Salt\SaltFactory::class);
```

Comment:
`Saltedpasswords` is deprecated, the correct class to instantiate and method to call should be `\TYPO3\CMS\Core\Crypto\PasswordHashing\PasswordHashFactory::getDefaultHashInstance`

Reply:
Changed in 698cfaa